frontend/tests/e2e-tests.sh

```
@@ -183,26 +181,28 @@ export DEFAULT_FROM_EMAIL='ciso-assistant@tests.net'
 export EMAIL_HOST=localhost
 export EMAIL_HOST_PASSWORD=pwd
 export EMAIL_PORT=$MAILER_SMTP_SERVER_PORT
+export CISO_ASSISTANT_VERSION=$(git describe --tags --always)
+export CISO_ASSISTANT_BUILD=$(git rev-parse --short HEAD)
 
 cd $APP_DIR/backend/
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)